main.js

```diff
@@ -1,14 +1,34 @@
 import AdderRemovebooks from './JS-Classes/AdderRemovebooks.js';
 import ListBooks from './JS-Classes/listBook.js';
+import Navigation from './JS-Classes/Navigation.js';
 
 const adderRemovebooks = new AdderRemovebooks();
 const listBooks = new ListBooks();
+const nav = new Navigation();
+// const datetime = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
```

Comment:
- [ OPTIONAL ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: